superbench/common/utils/network.py

```diff
@@ -31,6 +32,8 @@ def get_ib_devices():
     Return:
         ib_devices_port (list): IB devices with available ports in current system.
     """
+    if os.getenv('IB_DEVICES', None):
+        return os.getenv('IB_DEVICES').split(',')
```

Comment:
this function returns a list of `ib:port`, not just a list of `ib`. seems this change is not compatible with previous function

Reply:
we want to use the env to override this function(network.get_ib_devices) for both ib_validation and ib_loopback for some special use case like roce devices or gb200 bm, and IB_DEVICES env still can accept both ib1,ib2,ib3 or ib1:port1,ib2:port2,ib3:port3